analyzer/src/test/kotlin/managers/utils/MavenDependencyHandlerTest.kt

```diff
@@ -158,7 +158,9 @@ class MavenDependencyHandlerTest : WordSpec({
             val handler = createHandler(sbtMode = true)
 
             every { dependency.repositories } returns repos
-            every { handler.support.parsePackage(artifact, repos, sbtMode = true) } returns pkg
+            every {
+                handler.support.parsePackage(artifact, repos, sbtMode = true)
+            } returns pkg
```

Comment:
Unrelated refactoring (I'd actually keep the one-liner).

Reply:
Undone.